packages/vue/src/types/index.ts

```diff
@@ -1,3 +1,5 @@
+import { ComputedRef, InjectionKey, Ref, Slot } from 'vue';
+import { Interpreter } from 'xstate';
 import {
```

Comment:
nit: we can add a line break between internal and external dependencies 😉

Reply:
Will do in clean up PR (will happen the next one)!